src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/CalculateContamination.java

```diff
@@ -57,17 +63,30 @@
 
```

Comment:
Can we have some docs (does not have to be here) about how the algorithm works?

Reply:
Done in mutect2.pdf

Reply:
Right, forgot...